server/utils/sites.js

```diff
@@ -0,0 +1,41 @@
+export const SITE_NAMES = {
```

Comment:
should this file be named `siteNames.js`? or are there other site related constants that will go here later?

Reply:
Makes sense, thanks
https://github.com/AMP-SCZ/dpdash/pull/246/commits/9d9d1b15c3134f5ae991e11553bcddefde75cb55